addons/controllers/certificates/certificates.go

```diff
@@ -0,0 +1,159 @@
+package certificates
```

Comment:
I agree with @vijaykatam's suggestion that this should be in a package outside `addons`. I don't know if that's necessarily `pkg/v1/sdk/controllers`, but I guess it's better than `pkg/v1/utils`?

cc @jmoroski @yharish991

Reply:
done. Moved to pkg/v1/webhooks/certificates.go